learning_table_tennis_from_scratch/models.py

```diff
@@ -41,28 +47,39 @@ def run_stable_baselines(
     }
     env = make_vec_env(HysrOneBallEnv, env_kwargs=env_config)
 
-    model = PPO(
-        "MlpPolicy",
-        env,
-        seed=seed,
-        policy_kwargs={
-            "net_arch": [ppo_config.num_hidden] * ppo_config.num_layers,
-        },
-        **ppo_config.get_ppo_params(),
-    )
+    if algorithm == "ppo":
+        model = PPO(
+            "MlpPolicy",
+            env,
+            seed=seed,
+            policy_kwargs={
+                "net_arch": [rl_config.num_hidden] * rl_config.num_layers,
+            },
+            **rl_config.get_rl_params(),
+        )
+    else:
+        model = SAC(
+            "MlpPolicy",
+            env,
+            seed=seed,
+            policy_kwargs={
+                "net_arch": [rl_config.num_hidden] * rl_config.num_layers,
+            },
+            **rl_config.get_rl_params(),
+        )
```

Comment:
As long as all algorithms take the exact same arguments, you could do something like this to avoid code duplication:
```
model_types = {"ppo": PPO, "sac": SAC}
model = model_types[algorithm](
    "MlpPolicy",
    env,
    ...
)
```